### ISSUE TITLE
Bunch of broken entries in the list

### ISSUE THREAD

**Issue by Zenexer** [plaintextoffenders/plaintextoffenders#13]
Some of these appear to be the result of parsing errors.  Others look intentional but result in difficult-to-parse entries that are unlikely to be handled correctly by automated systems.

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L28

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L44-L46

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L99

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L128

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L130-L131

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L138-L140

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L266

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L305

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L428

https://github.com/plaintextoffenders/plaintextoffenders/blob/dd1a951b9b4092d1c54f18b92a51e85ad52fa255/offenders.csv#L468

**Comment by aviemzur** [comment]
Yeah, I tried to fix most of them that followed a pattern, but some still remain, especially in the beginning of the file.

**Comment by RokeJulianLockhart** [comment]
> Yeah, I tried to fix most of them that followed a pattern, but some still remain, especially in the beginning of the file.

@aviemzur, did you submit a PR?
